crates/bevy_ecs/src/schedule/config.rs

```diff
@@ -323,8 +323,20 @@ where
     /// If automatically inserting [`apply_deferred`](crate::schedule::apply_deferred) like
     /// this isn't desired, use [`after_ignore_deferred`](Self::after_ignore_deferred) instead.
     ///
-    /// Note: The given set is not implicitly added to the schedule when this system set is added.
-    /// It is safe, but no dependencies will be created.
+    /// Usually, you'll want to use [`.chain`](Self::chain) instead.
+    /// # Caveats
+    ///
+    /// If you configure two [`SystemSet`]s like `(GameSystem::A).after(GameSystem::B)` or `(GameSystem::A).before(GameSystem::B)`, the `GameSystem::B` will not be automatically scheduled.
+    ///
+    /// This means that the systems in `GameSystem::A` and `GameSystem::B` will run independently of each other if `GameSystem::B` was never explicitly scheduled with [`App::configure_sets`].
```

Comment:
I saw that in my editor 
```rust
331:  ... scheduled with [`App::configure_sets`].
```

Why is that a problem. I thought the comment was intended like this.

Reply:
Because `bevy_ecs` doesn't know about `App`. We usually fix that by just linking to `docs.rs`. Let me help you, sec.